certified-connectors/Public360/apiDefinition.swagger.json

```diff
@@ -0,0 +1,388 @@
+{
+   "swagger":"2.0",
+   "info":{
+      "version":"1.0.0",
+      "title":"Public 360",
+      "description":"Public 360 is a market leading ECM system in the Nordics. The Public 360 connector enables users to create flows that archive cases, documents and files into Public 360. The Public 360 connector will enable users to archive more and easier! That includes archiving social media posts, Outlook items or OneDrive documents.",
+      "contact":{
+         "name":"360 support",
+         "url":"https://tieto.service-now.com/",
+         "email":"360ServiceSupport@tieto.com"
+      }
+   },
+   "host":"paspublicnorlrg.dev.360online.com",
+   "basePath":"/",
+   "schemes":[
+      "https"
+   ],
+   "consumes":[
+      
+   ],
+   "produces":[
+      "application/json"
+   ],
+   "paths":{
+      "/Biz/v2/api/call/SI.Data.RPC/SI.Data.RPC/FileService/CreateFile":{
+         "post":{
+            "summary":"This operation is used to create a file in Public 360",

```

Comment:
@Pranav2889 The summary value is like a title of the action while creating flows you can that, this summary value looks more like a description rather than a title of the action. You can refer [this](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/CompanyConnect/apiDefinition.swagger.json#L363) as an example to improve all the summary values in the swagger

Reply:
Updated.